lib/version_utils.pm

```diff
@@ -875,6 +875,7 @@ sub get_bootloader {
     return $bootloader if $bootloader;
 
     return 'grub2' if !check_var('UEFI', 1);
+    return 'grub2-bls' if is_upgrade && !is_microos && (get_var('HDD_1') =~ /Tumbleweed/);
```

Comment:
Could we make this the default by removing `return 'grub2' if is_upgrade;` and instead setting `BOOTLOADER=grub2` for affected upgrade tests?

I rather dislike matches based on HDD_1 filenames

Reply:
Done, affected tests now have the `BOOTLOADER=grub2` and grub2-bls is the default for upgrade tests